test/__init__.py

```diff
@@ -74,6 +78,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def only_with_brotlipy():
```

Comment:
Let's keep the semantics of `notX` and `requiresX` consistent.

Reply:
Sure